Marlin/src/gcode/calibrate/G28.cpp

```diff
@@ -205,9 +205,9 @@ void GcodeSuite::G28(const bool always_home_all) {
         #endif
       }
 
-    #else
+      else
 
-      if (home_all || homeX || homeY) {
+      if (homeX || homeY) {
         // Raise Z before homing any other axes and z is not already high enough (never lower z)
```

Comment:
What a nonsense!
I really love it when my machine can home when z is not homing to max.